recipes/h5cpp/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class H5cppConan(ConanFile):
+    name = "h5cpp"
+    version = "0.4.1"
+    license = "LGPL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ess-dmsc/h5cpp"
+    description = "h5cpp is a new C++ wrapper for HDF5s C-API"
+    topics = ("HDF5",)
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _has_std_filesystem(self):
+        minimal_cpp_standard = "17"
+        if self.settings.compiler.cppstd:
+            return tools.valid_min_cppstd(self, minimal_cpp_standard)
+
+        # Information from https://en.cppreference.com/w/cpp/compiler_support
+        minimal_version = {
+            "gcc": "8",
+            "clang": "7",
+            "apple-clang": "11",
+            "Visual Studio": "19.14"
```

Comment:
I do not even know if this is allowed in CCI but the compiler version does not exist https://docs.conan.io/en/1.36/reference/config_files/settings.yml.html

```suggestion
            "msvc": "19.14"
```